microcloud/service/lxd.go

```diff
@@ -48,30 +48,25 @@ func (s LXDService) Bootstrap() error {
 	profile := api.ProfilesPost{ProfilePut: api.ProfilePut{Devices: rootDisk}, Name: "default"}
 	storage := api.StoragePoolsPost{Name: "local", Driver: "dir"}
 
-	initData := api.InitPreseed{
-		Node: api.InitLocalPreseed{
-			ServerPut:    server,
-			StoragePools: []api.StoragePoolsPost{storage},
-			Profiles:     []api.ProfilesPost{profile},
-		},
+	initData := initDataNode{
```

Comment:
@masnax out of interest, why not use api.InitLocalPreseed here? They look the same?